uniflow-test/src/test/kotlin/io/uniflow/test/StackFlowTest.kt

```diff
@@ -115,34 +107,35 @@ class StackFlowTest {
         dataFlow.add("first")
         dataFlow.makeOnError()
 
-        dataFlow.hasStates(
+        dataFlow.assertReceived(
                 UIState.Empty,
                 TodoListState(emptyList()),
                 TodoListState(listOf(Todo("first"))))
-        assert(dataFlow.lastEvent is UIEvent.Fail)
+        assert(dataFlow.lastEvent is UIEvent.Error)
```

Comment:
I see that you use `assert` quite a lot, instead of `assertTrue`. Are you sure (and do you have a source) that JVM assertions are always enabled? Otherwise these calls might not run and therefore failure could be missed.

Reply:
assert is the Kotlin assertion tool

Reply:
I know, but look at its implementation: it simply uses the JVM assertions!

Reply:
or we may depend from JUnit then, we embed assert tools then